include/emp/prefab/Card.hpp

```diff
@@ -27,29 +89,50 @@ namespace prefab {
 
   public:
     /**
-     * @param state indicate whether card should be STAITC, INIT_OPEN, or INIT_CLOSED (default STAITC)
+     * @param state indicate whether card should be STATIC, INIT_OPEN, or INIT_CLOSED (default STATIC)
      * @param show_glyphs should toggle icons show in collapsible card header? (default true)
      * @param id user defined ID for card Div, (default emscripten generated)
      */
     Card(
       const std::string & state="STATIC",
       const bool & show_glyphs=true,
       const std::string & id=""
-    ): web::Div(id) {
+    ) : Card(state, show_glyphs, new internal::CardInfo(id)) { ; }
+
+  protected:
+    /**
+     * A protected constructor for a Card.
+     * @param state indicate whether card should be STATIC, INIT_OPEN, or INIT_CLOSED (default STATIC)
+     * @param show_glyphs should toggle icons show in collapsible card header? (default true)
+     * @param info_ref a pointer to the underlying ReadoutPanelInfo object for this ReadoutPanel
+     * or a pointer to a derived info object (simulating inheritance)
+     */
+    Card(
+      const std::string & state,
+      const bool & show_glyphs,
+      internal::CardInfo * info_ref
+    ) : Div(info_ref) {
 
       AddBootstrap();
       if (state == "STATIC") { // static card with no toggle enabled
-        ((emp::web::Div) *this) << card_header; // Div cast prevents trying to add to body
-        ((emp::web::Div) *this) << card_body;
+        static_cast<Div>(*this) << card_header; // Div cast prevents trying to add to body
```

Comment:
I think I understand what you’re saying, but say it more descriptively than “trying to add to”